Core/include/Acts/Propagator/StandardAborters.hpp

```diff
@@ -161,23 +164,28 @@ struct SurfaceReached {
 
       // reaching the target calls a navigation break
       navigator.targetReached(state.navigation, true);
-    } else {
-      // Target is not reached, update the step size
-      const double overstepLimit = stepper.overstepLimit(state.stepping);
-      // Check the alternative solution
-      if (distance < overstepLimit and sIntersection.alternative) {
-        // Update the distance to the alternative solution
-        distance = sIntersection.alternative.pathLength;
-      }
-      stepper.setStepSize(state.stepping, distance, ConstrainedStep::aborter,
```

Comment:
Just to confirm, we don't need to handle this case anymore, because the intersections will both be in the candidate list, right?

Reply:
right this should be equivalent to the loop below